_overviews/toolkit/munit-assertions.md

```diff
@@ -1,18 +1,218 @@
 ---
 title: How to write assertions?
 type: section
-description: ???
+description: Comparing what you obtain with what you expect
 num: 4
 previous-page: munit-test-suite
 next-page: munit-exceptions
 ---
 
 {% include markdown.html path="_markdown/install-munit.md" %}
 
-<!--
-Tutorial about `assert`, `assertEquals`, `assertNotEquals` and `assertNoDiff`
+Assertions describe what to check in your tests. If any assertion in a unit test
+fails, then the test fails.
 
-https://scalameta.org/munit/docs/assertions.html
+Assuming you have defined a
+[test suite]({% link _overviews/toolkit/munit-test-suite.md %}) `MyTests`, this
+tutorial shows various ways to write assertions within test specifications.
 
-We explain clues in a later tutorial.
- -->
\ No newline at end of file
+{% tabs assertions-0 class=tabs-scala-version %}
+{% tab 'Scala 2' %}
+```scala
+class MyTests extends munit.FunSuite {
+  test("a unit test") {
+    // test specification goes here
+  }
+}
+```
+{% endtab %}
+{% tab 'Scala 3' %}
+```scala
+class MyTests extends munit.FunSuite:
+  test("a unit test") {
+    // test specification goes here
+  }
+```
+{% endtab %}
+{% endtabs %}
```

Comment:
While I agree we should not repeat this every where, I think it would be better to move it to the next example. That's because, as a user, the first block I want to see is the solution of the problem.

```
For instance, assuming your program provides a method `double`, ..., you can write the following test:

class MyTests extends munit.FunSuite {
  test("a unit test") {
    assertEquals(double(21), 42)
  }
}

This test contains the assertion: `assertEquals(double(21), 42)
```

Reply:
Removed in 23521c6